README.md

```diff
@@ -14,6 +14,20 @@ use({
     'nvim-neotest/neotest-go',
   }
   config = function()
+    -- get neotest namespace (api call creates or returns namespace)
+    local neotest_ns = vim.api.nvim_create_namespace("neotest")
+    vim.diagnostic.config({
+    	virtual_text = {
+    		format = function(diagnostic)
+    			local message = diagnostic.message
+    				:gsub("\n", " ")
+    				:gsub("\t", " ")
+    				:gsub("%s+", " ")
+    				:gsub("^%s+", "")
+    			return message
+    		end,
+    	},
+    }, neotest_ns)
```

Comment:
@jens1205 is the correct behaviour to create/recreate the neotest namespace or to re-use it? i.e. should the user be doing `api.nvim_get_namespaces()['neotest]` or creating it from scratch. @rcarriga what is the recommended usage here? will it have adverse effects to re-create the neotest namespace?

Reply:
Actually, just discovered that `nvim_create_namespace` acts as a get if the name is already in use so doesn't need a separate get here

Reply:
Sorry totally missed this :facepalm: Yes that's correct, it's a weird name :shrug: